client/src/components/Sidebar/index.jsx

```diff
@@ -23,27 +25,77 @@ const Sidebar = () => {
 	});
 
 	return (
+		<>
+		
+			<Header />
 		<div className={sidebarClass}>
 			{!isHidden && (
 				<>
-					<div>
-						<h1>
+					{/* <div className={styles.heading}>
+						<div className={styles.logo}>
 							<Image
-								className={styles.logo}
 								src={logoLight}
-								width="100"
-								height="30"
+								width={150}
+								height={30}
 								alt="icon"
 							/>
-							Kanban
-						</h1>
-					</div>
+						</div>
+						<h1 className={styles.headingText}>kanban</h1>
+					</div> */}
+
 					<span className={styles.allBoards}>ALL BOARDS</span>
+
 					<div className={styles.linkSection}>
-						<a href="#">Platform Launch</a>
-						<a href="#">Marketing Plan</a>
-						<a href="#">Roadmap</a>
+
+						<a href="#" className={styles.linkContainerA}>
+							<Image
+								className={styles.icon}
+								src={fluentBoard}
+								width={50}
+								height={20}
+								alt="icon"
+							/>
+							<p className={styles.link}>Platform Launch</p>
+						</a>
+
+						<a href="#" className={styles.linkContainerB}>
+							<Image
+								className={styles.icon}
+								src={fluentBoard}
+								width={50}
+								height={20}
+								alt="icon"
+							/>
+							<p className={styles.link}>Marketing Plan</p>
+						</a>
+
+						<a href="#" className={styles.linkContainerC}>
+							<Image
+								className={styles.icon}
+								src={fluentBoard}
+								width={50}
+								height={20}
+								alt="icon"
+							/>
+							<p className={styles.link}>Roadmap</p>
+						</a>
+
+						
+						<a href="#" className={styles.newBoardLinkCon}>
+							<Image
+								className={styles.newBoardLinkIcon}
+								src={fluentBoard2}
+								width={30}
+								height={15}
+								alt="icon"
+							/>
+							<p className={styles.newBoardLink}>+ Create New Board</p>
+						</a>
+						
+						
```

Comment:
Apologies, it seems that we have the create new board link existing here: https://github.com/Erikang20/Kanban-task-management-web-app/blob/main/client/src/components/Sidebar/index.jsx#L51

Reply:
This task wasn't created when I picked my task to work on It.